pubspec.yaml

```diff
@@ -13,9 +13,10 @@ dependencies:
   push:
     path: packages/push/push
   lemmy_api_client:
-    git:
-      url: "https://github.com/thunder-app/lemmy_api_client.git"
-      ref: 874005c9013de53f526562f882fe8231eb0ce5ae
+    path: ../lemmy_api_client
+    #git:
+    #  url: "https://github.com/thunder-app/lemmy_api_client.git"
+    #  ref: 16d14a1c13ac9522e85188ad9cf23d8912ec8fee
```

Comment:
Should merge this first https://github.com/thunder-app/lemmy_api_client/pull/28
And then update this.

Reply:
For now can you update this to point at your fork/branch? That will make building and testing easier. Then we can point it back to `thunder-app/lemmy_api_client` before merging.